controllers/application_controller.go

```diff
@@ -86,6 +90,25 @@ func (r *ApplicationReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		return reconcile.Result{}, err
 	}
 
+	// If the resource still has the finalizer attached to it, just remove it so deletion doesn't get blocked
+	if containsString(application.GetFinalizers(), appFinalizerName) {
```

Comment:
strings.Contains(application.GetFinalizers(), appFinalizerName)

Reply:
Just copied the code back to how we initially did it